### PR TITLE
Wizards can now cast iron

### PR DESCRIPTION
**Pull request by Moltijoe**
Shoots a projectile that does 20 damage and applies 4 seconds of confusion and stutter

# Why is this good for the game?
more silly wizard spells

# Testing
https://github.com/yogstation13/Yogstation/assets/108117184/261d1f21-6919-4d4c-b507-6c3c07101df2

:cl:  
rscadd: Wizards can now cast iron
/:cl:

### CODE REVIEW

**Comment by cowbot92** [comment]
Ok but make the level 5 version spew out bananium pans

**Comment by Moltijoe** [comment]
> Ok but make the level 5 version spew out bananium pans

that's a very dangerous train of thought

**Comment by cowbot92** [comment]
> > Ok but make the level 5 version spew out bananium pans
> 
> that's a very dangerous train of thought

I like trains

**Comment by Yarinoi** [comment]
wait what if it was like a weaker ranged version of the buster arm dash

**Comment by UltraBigRat** [comment]
Seems pretty weak for a wizard spell

File: code/modules/antagonists/wizard/equipment/spellbook_entries/offensive.dm [hunk 1]
```diff
@@ -5,6 +5,13 @@
 	spell_type = /datum/action/cooldown/spell/pointed/projectile/fireball
 	category = "Offensive"
 
+/datum/spellbook_entry/cast_iron
+	name = "Cast iron"
```
Comment (tattax) [inline code/modules/antagonists/wizard/equipment/spellbook_entries/offensive.dm]: pls follow spellbook naming convention

File: code/modules/spells/spell_types/pointed/cast_iron.dm [hunk 2]
```diff
@@ -0,0 +1,42 @@
+/datum/action/cooldown/spell/pointed/projectile/cast_iron
+	name = "Cast iron bolt"
```
Comment (tattax) [inline code/modules/spells/spell_types/pointed/cast_iron.dm]: .

**Pull request merged**